generators/generate.py

```diff
@@ -7,13 +7,19 @@
 import logging
 from argparse import ArgumentParser
 from collections import OrderedDict
-from pathlib import Path
+from functools import partial
 from itertools import repeat
+from pathlib import Path
 from string import punctuation, whitespace
 
 # 3rd party imports:
 from jinja2 import Environment, PackageLoader, select_autoescape, TemplateError
 from yapf.yapflib.yapf_api import FormatFile, style
+from yaml import load, dump
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except ImportError:
+    from yaml import Loader, Dumper
```

Comment:
I'm guessing this is for Python2/3 compatibility. Would you mind adding comments specifying which is which?

Reply:
Ah, of course. I should've taken that hint from the naming schemes. Still would you mind adding a comment explaining that for easy future reference?

Reply:
Sure.